src/main/kotlin/io/livekit/server/EgressServiceClient.kt

```diff
@@ -8,6 +8,12 @@ import retrofit2.Call
 import retrofit2.Retrofit
 import retrofit2.converter.protobuf.ProtoConverterFactory
 
+data class EncodedOutputs(
```

Comment:
Is it fine if multiple outputs are specified at the same time?

Reply:
yeah it allows up to one of each type now